src/frontend-samples/classifier-sample/ClassifierApi.ts

```diff
@@ -0,0 +1,106 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Update README to say that this file has the necessary static methods for interacting with iModel.js

Reply:
Readme update to point to proper file.